images/gateway-deployment/install.sh

```diff
@@ -24,19 +24,11 @@ usage() {
 
     # Parameters
     printf "The following parameters need to be passed to the script:\n"
-    printf "KCP_ORG: the organistation for which the workload clusters need to be registered, i.e.: root:pipeline-service\n"
```

Comment:
This option might be useful, I would probably keep it, but bypass `switch_org()` if unset.